share/tools/web_config/webconfig.py

```diff
@@ -49,6 +49,10 @@ def is_wsl():
                 return True
     return False
 
+def is_termux():
+    """ Return whether we are running under the Terumx application for Android"""
+    return 'com.termux' in os.environ['PATH'] and 0 == os.system('which termux-open-url 1>&2 2>/dev/null')
```

Comment:
`which termux-open-url 1>&2 2>/dev/null` will send stdout to stderr and stderr to `/dev/null`.
If you want to discard both, swap the order of the redirection operators.

Reply:
fixed

Reply:
Also this check itself is really quite.... unclean, and invokes another shell *and* the `which` command, which doesn't have to be installed and should be discouraged (in shell scripts, use `command -v`).

I'd suggest `distutils.spawn.find_executable('termux-open-url')`, which returns a path if it finds it and None if it doesn't.

Reply:
using `distutils` instead of `which`